client/server/pages/index.js

```diff
@@ -322,10 +322,10 @@ function setUpLoggedInRoute( req, res, next ) {
 				const end = new Date().getTime() - start;
 
 				debug( 'Rendering with bootstrapped user object. Fetched in %d ms', end );
-				req.context.user = data;
+				req.context.user = { ...data, bootstrapped: true };
```

Comment:
I wonder if it makes sense to add this field in `getBootstrappedUser()` instead. While it won't make a huge difference right now, since the only usage of `getBootstrappedUser()` is here, we might end up using it somewhere else, and then it could be useful to have the `bootstrapped` field there. What do you think?

Reply:
That makes a lot of sense, I'll change it.